yarn-project/merkle-tree/src/merkle_tree.ts

```diff
@@ -35,6 +36,16 @@ export interface MerkleTree extends SiblingPathSource {
    * Commit pending updates to the tree
    */
   commit(): Promise<void>;
+  /**
+   * Updates a leaf at a given index in the tree
+   * @param leaf The leaf value to be updated
+   * @param index The leaf to be updated
+   */
+  updateLeaf(leaf: Buffer | LeafData, index: bigint): Promise<void>;
```

Comment:
```suggestion
  updateLeaf(leaf: LeafData, index: bigint): Promise<void>;
```

At least from looking at other functions, right?

Reply:
Correct